.github/workflows/ci.yaml

```diff
@@ -0,0 +1,35 @@
+name: Go test
+
+on:
+  pull_request:
+
+jobs:
+  run-test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.19' ]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go }}
+          cache: true
+        
+      - name: Install goimports
+        run: go get golang.org/x/tools/cmd/goimports
+
+      - name: Test
+        run: go mod tidy
+
+      - name: Test
+        run: go test -v ./...
+
+      - name: Test
+        run: go vet
+    
+      - name: goimports
+        run: test -z "$(set -o pipefail && $(go env GOPATH)/bin/goimports -l . | tee goimports.out)" || { cat goimports.out && exit 1; }
```

Comment:
I think `setup-go@v3` actually adds GOBIN to the PATH (based on these docs https://github.com/actions/setup-go#v3)

So you can likely just call `goimports` (provided you also use `go install` as the previous comment suggested).